torch/csrc/autograd/engine.cpp

```diff
@@ -1262,14 +1262,7 @@ void GraphTask::stash_current_streams() {
   caller_current_streams_.resize(num_gpus);
   if (num_gpus > 0) {
     for (c10::DeviceIndex idx = 0; idx < num_gpus;  idx++) {
-#if defined(USE_ROCM)
```

Comment:
@rraminen , need to key off ROCm version >= 5.0

Reply:
Added the  ROCm version check conditions to torch/csrc/autograd/engine.cpp.

Yet to add it to skip the TestCudaPrimaryCtx tests in test/test_cuda_primary_ctx.py.